### ISSUE TITLE
How to force upgrade from the old format for encryption?

### ISSUE THREAD

**Issue by albill** [jrnl-org/jrnl#899]
## Support request

I had a journal that was running with jrnl 1.9.8 on OS X. I used Homebrew and updated to 2.2 and received the prompt to upgrade my journal with the new encryption scheme. This I did and it made a backup. 

This journal was on a dropbox. On a Linux system running 2.0.1, something happened and it failed to open the upgraded file and it wound up writing out a 100 byte file, overwriting my journal. 

I restored my backup to go through the upgrade process again but when I run jrnal 2.2 on my OS X machine, it fails to go through the upgrade prompt and, after three password attempts, tells me bad password and fails to decrypt the file.

How to I force the encryption upgrade process again since my jrnl file is from my version 1.9.8 version backup?

**Comment by micahellison** [comment]
Hi there, you should be prompted to upgrade again if you delete your jrnl v2.x configuration file at: `~/.config/jrnl/jrnl.yaml`

Once that's gone, jrnl should see your old configuration file and take you through the upgrade process again.

**Comment by albill** [comment]
Thanks. I have no idea if this would work but I'm glad it is noted here. I managed to roll back the version to 1.9.8, verified I could edit my old journal file, and then upgraded the installed version to current with Homebrew on Linux and pip3 on Linux. I've verified both are reading the file now.

It would be nice if the failure mode earlier hadn't resulted in writing an empty journal file on top of my old file location though!
